test/geometry_unittest.cpp

```diff
@@ -131,6 +139,14 @@ TEST_CASE("saber::geometry::size", "[saber]")
 		REQUIRE(size2.Width() == 1);
 		REQUIRE(size2.Height() == 2);
 	}
+	
+	SECTION("Structured Binding")
+	{
+		saber::geometry::Size<int> size{1,2};
+		const auto [height, width] = size; // accessor: structured binding!
+		REQUIRE(height == 1);
```

Comment:
this appears to be backwards...
Q: which argument is which, in the ctor{} above? Which is width vs. height?
Q: Given that, what is the ordering of the structured binding elements?